testhelpers/binaries.go

```diff
@@ -0,0 +1,117 @@
+package testhelpers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+)
+
+func MakeAndCopyLauncher(t *testing.T, destDir string) {
```

Comment:
Changing it to be consistent. FWIW the reason this didn't take a `goos` is because in practice when we're testing the launcher it's always either Linux or Windows, so `runtime.GOOS` + some logic to convert darwin -> linux is sufficient. For the lifecycle, because we test the version on darwin the function needed more flexibility. What I have now is explicitly passing daemon OS as the goos for the launcher.